File: src/day1/python/BubbleSort.py
def bubble_sort(arr: list[int]) -> None:
    length = len(arr)
    while length > 0:
        for i in range(length-1):
            if arr[i] > arr[i+1]:
                arr[i], arr[i+1] = arr[i+1], arr[i]
        length -= 1

File: src/day1/go/bubble_sort.go
package main

func BubbleSort(arr []int) {
	length := len(arr)
	for length > 0 {
		for i := 0; i < length-1; i++ {
			if arr[i] > arr[i+1] {
				arr[i], arr[i+1] = arr[i+1], arr[i]
			}
		}
		length--
	}
}
